ikasaneip/builder/jar/src/test/java/org/ikasan/builder/FlowBuilderTest.java

```diff
@@ -350,6 +352,7 @@ public void test_successful_simple_transitions()
 
         Assert.assertTrue("flow name is incorrect", "flowName".equals(flow.getName()));
         Assert.assertTrue("module name is incorrect", "moduleName".equals(flow.getModuleName()));
+        Assert.assertTrue("flow is recording", true == ((FlowPersistentConfiguration)((ConfiguredResource)flow).getConfiguration()).getIsRecording());
```

Comment:
Given the code I would suggest to put a test for the case when isRecording=false